src/plugins/liveobjects/statemessage.ts

```
@@ -197,6 +186,119 @@
+  private static _encodeStateOperation(
+    platform: typeof Platform,
+    stateOperation: StateOperation,
+    withBase64Encoding: boolean,
+  ): StateOperation {
+    // deep copy "stateOperation" object so we can modify the copy here.
+    // buffer values won't be correctly copied, so we will need to set them again explictly.
+    const stateOperationCopy = JSON.parse(JSON.stringify(stateOperation)) as StateOperation;
+
+    if (stateOperationCopy.mapOp?.data && 'value' in stateOperationCopy.mapOp.data) {
+      // use original "stateOperation" object when encoding values, so we have access to the original buffer values.
+      stateOperationCopy.mapOp.data = this._encodeStateData(platform, stateOperation.mapOp?.data!, withBase64Encoding);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)